### PR TITLE
EWL-4899 input error

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4899: SG2 | Create "Text field as Errored" atom" atom](https://issues.ama-assn.org/browse/EWL-4899)

## Description
Create input field atom in error state.

## To Test
- [ ] `gulp serve`
- [ ] visit http://localhost:3000/?p=atoms-text-field-as-error
- [ ] Observe label that says LABEL
- [ ] Observe an input field with "Text field placeholder" for a placeholder
- [ ] Observe the help text below the input
- [ ] Observe that the input box has a red border
- [ ] Observe the red text below the input box

## Visual Regressions
backstop test only noted the footer organism as failures due to it shifting positions.

## Relevant Screenshots/GIFs
screen shot 2018-04-24 at 3 21 10 pm

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/01-atoms/_text-field.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+input[type="datetime-local"],
+input[type="week"],
+input[type="number"],
+input[type="search"],
+input[type="tel"],
+input[type="color"],
+select,
+textarea {
+  /* stylelint-disable property-no-vendor-prefix */
+  -webkit-appearance: none;
```
Comment (labbydev) [inline styleguide/source/assets/scss/01-atoms/_text-field.scss]: Shouldn't auto prefixer be adding this?

**Review by labbydev — COMMENTED** [review]
Just a quick question about having to use the vendor prefixes

File: styleguide/source/assets/scss/01-atoms/_text-field.scss (hunk 1 above)
Comment (designerneil) [reply styleguide/source/assets/scss/01-atoms/_text-field.scss]: probably. will remove

**Comment by nstriedinger** [comment]
@designerneil Should these form elements be 100% width to fill out their container?

**Pull request merged**